### PR TITLE
scfix behaviour change

### PR DESCRIPTION
**Pull request by csbrasnett**
opening this as a PR for changing the default for `-scfix`.

Obviously atm this makes a bunch of tests fail. Is the best way to fix the test to:

1) change the command being used as the input
2) change the expected output of the test

or both?

Do we change the flag description too, to something like -[no]scfix, ie. when the flag is applied scfix gets turned off?

### CODE REVIEW

**Comment by csbrasnett** [comment]
Unfortunately, I think we'll have to change (at least some of) the tests. I've noticed a couple of the tests don't actually reflect the default applications. I don't think it should take too long to do :crossed_fingers:

File: bin/martinize2 [hunk 1]
```diff
@@ -628,9 +628,9 @@ def entry():
     prot_group.add_argument(
         "-scfix",
         dest="scfix",
-        action="store_true",
+        action="store_false",
         default=True,
-        help="Apply side chain corrections.",
+        help="[don't] apply side chain corrections.",
     )
```
Comment (pckroon) [inline bin/martinize2:634]: So, the name of the flag remains the same, but the behaviour is inverted? I'm not sure that's very insightful

File: vermouth/tests/data/integration_tests/tier-0/mini-protein1_betasheet/martinize2/command [hunk 2]
```diff
@@ -5,3 +5,5 @@ martinize2
 -ff martini22
 -dssp
 -cys auto
+-ef 2500
```
Comment (pckroon) [inline vermouth/tests/data/integration_tests/tier-0/mini-protein1_betasheet/martinize2/command:8]: Typo?

**Review by pckroon — CHANGES_REQUESTED** [review]

File: bin/martinize2 (hunk 1 above)
Comment (csbrasnett) [reply bin/martinize2]: change name to `-noscfix`?

Comment (pckroon) [reply bin/martinize2]: That would work for me.

**Review by pckroon — CHANGES_REQUESTED** [review]
Yes, excellent!
Just some teenie tiny nitpicks

**Comment by pckroon** [comment]
Probably because 9.0 is not more than 9.0 or something dumb. We need to update the pylint config file as well it seems, lots of deprecated options.
Doesn't really matter at the moment though.
Excellent work!

**Comment by csbrasnett** [comment]
aha thanks! I'll have a look at #616 to check if that's ready to merge too and then we're set

**Pull request merged**